docs/docs/NEW_DOCS/aztec_nr/advanced/calling.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
if this is calling other functions in the contract - would be nice to not have it in `advanced`

Same for `partial notes`, `events` and `authwits` -> these are needed for virtually every function!